backend/v3/domain/org_update.go

```diff
@@ -0,0 +1,66 @@
+package domain
+
+import (
+	"context"
+
+	"github.com/zitadel/zitadel/backend/v3/storage/database"
+	"github.com/zitadel/zitadel/internal/api/authz"
+	"github.com/zitadel/zitadel/internal/zerrors"
+)
+
+type UpdateOrgCommand struct {
+	ID   string `json:"id"`
+	Name string `json:"name"`
+}
+
+var _ Commander = (*UpdateOrgCommand)(nil)
+
+func (u *UpdateOrgCommand) Execute(ctx context.Context, opts *CommandOpts) (err error) {
+	close, err := opts.EnsureTx(ctx)
+	if err != nil {
+		return err
+	}
+	defer func() { err = close(ctx, err) }()
+
+	organizationRepo := orgRepo(opts.DB)
+
+	updateCount, err := organizationRepo.Update(
+		ctx,
+		organizationRepo.IDCondition(u.ID),
+		authz.GetInstance(ctx).InstanceID(),
+		database.NewChange(organizationRepo.NameColumn(), u.Name),
+	)
+	if err != nil {
+		return err
+	}
+
+	if updateCount == 0 {
+		err = zerrors.ThrowNotFound(nil, "DOM-7PfSUn", "organization not found")
```

Comment:
would it make sense to create fixed errors in a separate errors.go file analog to https://github.com/zitadel/zitadel/blob/feat/10445-organization-apis-with-rel-tables/backend/v3/storage/database/errors.go#L1?

Reply:
Yes, I can do that